governance/xc-admin/packages/xc-admin-common/src/wormhole.ts

```diff
@@ -9,3 +9,12 @@ export const WORMHOLE_ADDRESS: Record<PythCluster, PublicKey | undefined> = {
   localnet: new PublicKey("Bridge1p5gheXUvJ6jGWGeCsgPKgnE3YgdGKRVCMY9o"),
   testnet: undefined,
 };
+
```

Comment:
Let's just leave a comment here to this link if anyone wants to change it in the future:
https://book.wormhole.com/reference/rpcnodes.html